docker-compose.yml

```diff
@@ -21,44 +31,46 @@ services:
       - zookeeper
 
   influxdb:
-      image: influxdb:latest
+      image: influxdb:${INFLUXDB_VERSION:-1.7-alpine}
       container_name: influxdb
       ports:
           - "8086:8086"
       volumes:
           - ${PWD}/influxdb.conf:/etc/influxdb/influxdb.conf:ro
+          - influxdb:/var/lib/influxdb
 
   redis:
-      image: 'bitnami/redis:latest'
+      image: bitnami/redis:${REDIS_VERSION:-5.0.9}
       container_name: redis
       environment:
         - ALLOW_EMPTY_PASSWORD=yes
       ports:
         - '6379:6379'
 
   repose-public:
-    image: rackerlabs/repose:${REPOSE_VERSION:-9.0.0.0}
+    image: rackerlabs/repose:${REPOSE_VERSION:-9.0.1.0}
     ports:
       - 8180:8080
     env_file:
       - common.env
     environment:
       KEYSTONE_USER: ${KEYSTONE_USER}
       KEYSTONE_PASSWORD: ${KEYSTONE_PASSWORD}
-      BACKEND_HOST: ${REPOSE_BACKEND_HOST}
-      BACKEND_PORT: 8080
+      BACKEND_HOST: ${BACKEND_HOST:-host.docker.internal}
```

Comment:
@Yurochkina with this change you no longer need to lookup your computer's IP address and set `REPOSE_BACKEND_HOST`. The default will now automatically route to locally running query-service.

Reply:
That is very helpful ! Thank you @itzg